quickstart/speech-translation/cpp-windows/helloworld/helloworld.cpp

```diff
@@ -4,7 +4,6 @@
 //
 
 // <code>
-#include "pch.h"
```

Comment:
can you swap lines 6 and 7 instead of deleting line 7? This may cause a compilation break, from the sample repository perspective.